roles/nuage_node/handlers/main.yaml

```diff
@@ -6,3 +6,7 @@
 - name: restart node
   become: yes
   service: name={{ openshift.common.service_type }}-node state=restarted
+
+- name: save iptable rules
+  become: yes
+  command: iptables-save
```

Comment:
This command is not actually saving the iptables rules.  iptables-save just dumps the loaded tables to stdout.  Additionally, a wholesale saving of iptables rules has been discouraged by the OpenShift networking team.  A possible alternative is using the os_firewall role to create rules.  cc: @detiber @sdodson